src/ReservoirPriceOracle.sol

```diff
@@ -113,6 +113,7 @@ contract ReservoirPriceOracle is IPriceOracle, IReservoirPriceOracle, Owned(msg.
         (rRoute,,) = _getRouteDecimalDifferencePrice(aToken0, aToken1);
     }
 
+    // REVIEW: Is this intended primarily for debugging? Because this won't work on composite routes right?
```

Comment:
yes, mainly for auditability. Correct, won't work for composite routes, as documented

Reply:
Okay, can remove the comment